op-chain-ops/genesis/setters.go

```diff
@@ -89,3 +90,11 @@ func setupPredeploy(db vm.StateDB, deployResults immutables.DeploymentResults, s
 
 	return nil
 }
+
+// setup4788Contract creates the EIP-4788 beacon-block-roots contract, part of the Ecotone upgrade.
+func setup4788Contract(db vm.StateDB) error {
```

Comment:
We should really delete this function and instead just run the upgrade txs

Reply:
Implemented